editwatcher.py

```diff
@@ -0,0 +1,160 @@
+# coding=utf-8
+import json
+import os.path
+import requests
+import time
+import threading
+# noinspection PyPackageRequirements
```

Comment:
stylistic fix: add a space between standard library imports and third-party imports, and again between third-party imports and local imports

Reply:
I'm happy to use that style. It is, however, not something we're consistently using across SD. These lines are direcctly copied from the existing deletionwatcher.py file, upon which editwatcher.py was based.

Reply:
Yeah, I'm fine with leaving that in, and tackling minor code style in a separate PR eventually. I've been meaning to do that, idly.

Reply:
A separate PR would be good for applying a general style template.

Given that I did need to go through and remove dependencies in some files where the need for them had been eliminated, I went ahead and made the style adjustments you've described in several of the files which I had already edited for this PR. A PR with a global `import` style change to all files would probably be a good idea.